Octokit.Tests.Integration/Clients/RepositoryCollaboratorClientTests.cs

```diff
@@ -136,4 +136,26 @@ public class TheIsCollaboratorMethod
             }
         }
     }
+
+    public class TheInviteMethod
+    {
+        [IntegrationTest]
+        public async Task CanInviteNewCollaborator()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoName = Helper.MakeNameWithTimestamp("public-repo");
+
+            using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
+            {
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest();
+
+                // invite a collaborator
+                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, "maddin2016", permission);
```

Comment:
This test always fails because context is trying to delete this repo and you cannot delete a repo until there is a pending invitation. And yes, it still not sending any email. Next question is, how should i write integration tests for invitations. Is there any test client where user can send invitations to?